src/OptionalPackagesInstaller.php

```diff
@@ -15,6 +15,11 @@
 use Composer\Package\RootPackageInterface;
 use Composer\Package\Version\VersionParser;
 use Composer\Plugin\PluginInterface;
+
+use function call_user_func;
```

Comment:
Same comment as for the `ComposerJsonRetrievalTrait`, only this time, the function to update will be `runInstaller()`.

Reply:
Done